governance/reviews/cloudevents-08-29-2023.md

```diff
@@ -0,0 +1,210 @@
+| Code of Conduct | Complete | Uses link to CNCF CoC | |
+| Contributor Guide | Partial | [CONTRIBUTING](https://github.com/cloudevents/spec/blob/main/docs/CONTRIBUTING.md) | See discussion above |
+| Contributor Ladder | Partial | [GOVERNANCE](https://github.com/cloudevents/spec/blob/main/docs/GOVERNANCE.md) | Several roles exist, but qualifications and advancement are not explained |
+| Maintainer Lifecycle | Missing | | How to become, or retire, a Voting Member is not explained anywhere |
+| Decision-making | Complete | [GOVERNANCE](https://github.com/cloudevents/spec/blob/main/docs/GOVERNANCE.md), [SDK Governance](https://github.com/cloudevents/spec/blob/main/docs/SDK-GOVERNANCE.md) | Nonlinear, but explained in exacting detail |
+| Code and Docs Ownership | Complete | [OWNERS](https://github.com/cloudevents/spec/blob/main/OWNERS), plus each repo | |
+| Security Reporting and response | Missing | | A security team exists but is not documented |
+| Communication and Meetings | Partial | [GOVERNANCE](https://github.com/cloudevents/spec/blob/main/docs/GOVERNANCE.md) | meeting process, no calendar.  should link to SDK comms info |
+
+#### Sub-projects, plugins, and related
+
+<!--- If the project has subprojects, plugins, or other divisions define them here. For each, is ownership and operation of clearly described? Are any standing committees/teams fully described, including listing their members? Does it conform to, align, and is it within scope of the governance expectations of the project? --->
+
+The project includes the following sub-projects, plugins, and other notable divisions:
+
+| Area | Ownership and Operation | Standing Bodies | Project Alignment | Notes |
+|:-----|:-----------------------:|:---------------:|:------------------|:---|
+| SDK-Java | Partial | Complete | Partial | Currently a WIP project |
+| SDK-Csharp | Partial | Complete | Partial | |
+| SDK-Javascript | Partial | Complete | Partial | |
+| SDK-Rust | Missing | Complete | Partial | No maintainers listed |
+| SDK-Powershell | Missing | Missing | Partial | No maintainers, no meetings or channel |
+| SDK-PHP | Partial | Complete | Partial | No maintainers listed |
+| SDK-Ruby | Partial | Complete | Partial | No maintainers listed |
+
+Language-specific Software Development Kits (SDKs) exist as subprojects of CloudEvents.  These are largely autonomous of the main spec project in a governance sense.  There is a [template for SDK governance](https://github.com/cloudevents/spec/blob/main/docs/SDK-GOVERNANCE.md) and other guidance docs, but SDK projects are largely left on their own after creation.  This leads to some problems:
+
+* Some SDKs have gone completely unmaintained
+* There are no processes for SDK archiving or SDK Maintainer offboarding
+* No SDK roles are defined other than "maintainer"
+* Certification of SDKs is not up to date
```

Comment:
Which certification are you referring to ?

Reply:
Compliance with versions of the spec.  Several SDKs are not current.

Reply:
According to https://github.com/cloudevents/spec/blob/main/cloudevents/SDK.md#feature-support they all support v1.0 - which is sufficient since we're still in the v1.0.x track.